app/kumactl/pkg/cmd/root_context.go

```diff
@@ -102,16 +83,15 @@ func DefaultRootContext() *RootContext {
 	}
 }
 
-func (rc *RootContext) TypeForArg(arg string) (core_model.ResourceType, error) {
-	typ, ok := rc.TypeArgs[arg]
+func (rc *RootContext) TypeForArg(arg string, requiresWrite bool) (core_model.ResourceType, error) {
```

Comment:
We can just remove this, right? It's only used once and that caller can just look up the name in the entities map directly.